src/components/input/TextComponent.tsx

```diff
@@ -17,37 +17,47 @@ export type InputBaseProps = {
   onCopy?: (event) => any;
   onCut?: (event) => any;
   onDrag?: (event) => any;
-}
+};
 
 type TextComponentProps = {
+  /** React Element to display inside of the Field, on the right side */
+  decoratorElement?: JSX.Element; // what do you think about the name?
```

Comment:
I think it should be an array, wouldn't use element as it's redundant 
I'd say
rightDecorators: JSX.Element[]

Reply:
ok let's go for an array